Mail/Views/Switch User/AccountListView.swift

```diff
@@ -54,10 +54,8 @@ struct AccountListView: View {
                     }
                 }
             }
-
```

Comment:
You should still keep this empty lines!
Please do not resolve a conversation until it is fixed by a commit.